src/components/Visualizations/inflictorWithValue.jsx

```diff
@@ -39,7 +40,9 @@ const tooltipContainer = thing => (
       <div
         dangerouslySetInnerHTML={{
           __html: thing.cmb
-            .replace('/apps/dota2/images/tooltips/mana.png', `${API_HOST}/apps/dota2/images/tooltips/mana.png`)
+            .replace('src="/apps/dota2/images/tooltips/mana.png"', `src="${API_HOST}/apps/dota2/images/tooltips/mana.png"`)
```

Comment:
should we do this sort of transform in the `dotaconstants` code?

https://github.com/odota/dotaconstants/blob/master/tasks/updateconstants.js#L46